internal/apicodec/codec_v2.go

```diff
@@ -228,7 +228,8 @@ func (c *codecV2) EncodeRequest(req *tikvrpc.Request) (*tikvrpc.Request, error)
 		newReq.Req = &r
 	case tikvrpc.CmdBatchCop:
 		r := *req.BatchCop()
-		// TODO(iosmanthus): handle cop Request.
+		r.Regions = c.encodeCopRegions(r.Regions)
```

Comment:
This request is sent to TiFlash only, we may ignore this, for now, the tiflash requests should be sent in plain.